internal/db/schema/migrations/oss/postgres/42/05_key_rewrap.up.sql

```diff
@@ -0,0 +1,30 @@
+
+-- make the required schema changes to upgrade the dependency:
+-- github.com/hashicorp/go-kms-wrapping/extras/kms/v2 
+-- this migration is from:
+-- https://github.com/hashicorp/go-kms-wrapping/blob/main/extras/kms/migrations/postgres/05_key_rewrap.up.sql 
+
+begin;
+
+
+-- we need to make the key and version columns mutable in order to support
+-- rewrapping the root key versions.
+drop trigger kms_immutable_columns on kms_root_key_version;
+
+create trigger kms_immutable_columns
+before
+update on kms_root_key_version
+  for each row execute procedure kms_immutable_columns('private_id', 'root_key_id', 'create_time');
+
+
+
+-- we need to make the key and version columns mutable in order to support
+-- rewrapping the data key version.
+drop trigger kms_immutable_columns on kms_data_key_version;
+
+create trigger kms_immutable_columns
+before
+update on kms_data_key_version
+  for each row execute procedure kms_immutable_columns('private_id', 'data_key_id', 'root_key_version_id', 'create_time');
```

Comment:
Please reformat to follow our [sql style guide](https://github.com/hashicorp/boundary/blob/main/sql-style-guide.md#triggers).